modules/dataverse-exporter-spi/src/main/java/io/gdcc/export/spi/Exporter.java

```diff
@@ -43,8 +46,6 @@ but NOT close() it!
     public String getXMLSchemaLocation() throws ExportException; 
     public String getXMLSchemaVersion() throws ExportException; 
     
-    public void setParam(String name, Object value);
-
 	public default String getMediaType() {
 	    return MediaType.APPLICATION_XML;
```

Comment:
FWIW: I added an XMLExporter extends Exporter interface that gets rid of the three XML specific methods in the Exporter interface (which means that non-XML exporters don't have to implement and throw exceptions for them) and allows only XMLExporter to have a default value. It doesn't get rid of the MediaType dependency, but it might be good to have that so new exporters can use it.